examples/sleepy-demo/sleepy-mtd.c

```diff
@@ -58,8 +58,8 @@
 #define MULTICAST_PORT 123
 #define RECV_PORT 234
 #define SLEEPY_POLL_PERIOD_MS 2000
-#define MTD_MESSAGE "mtd button"
 #define FTD_MESSAGE "ftd button"
+#define MTD_MESSAGE "mtd button"
```

Comment:
This might cause conflicts since another PR also have the same/similar changes.

Reply:
This is an easy conflict to resolve. What PR are you referring to?

Reply:
Oh nevermind, you're referring to my sleepy-demo-ssed PR. It'll be fine. I can take care of the conflict if it comes up